velox/functions/sparksql/tests/JsonObjectKeysTest.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "velox/functions/sparksql/tests/SparkFunctionBaseTest.h"
+
+using namespace facebook::velox::test;
+
+namespace facebook::velox::functions::sparksql::test {
+namespace {
+
+class JsonObjectKeysTest : public SparkFunctionBaseTest {
+ protected:
+  VectorPtr jsonObjectKeys(std::optional<std::string> json) {
+    std::optional<StringView> s = json.has_value()
+        ? std::make_optional(StringView(json.value()))
+        : std::nullopt;
+    auto varcharVector = makeNullableFlatVector<StringView>({s}, VARCHAR());
+    auto varcharResult =
+        evaluate("json_object_keys(c0)", makeRowVector({varcharVector}));
+    return varcharResult;
+  }
+};
+
+TEST_F(JsonObjectKeysTest, baseTest) {
+  auto expected1 =
```

Comment:
Perhaps rename it as expected and reuse this variable.

Reply:
Done, thanks